tests/e2e/kserve_test.go

```diff
@@ -47,7 +47,7 @@ func kserveTestSuite(t *testing.T) {
 	t.Run("Validate default certs", componentCtx.validateDefaultCertsAvailable)
 	t.Run("Validate update operand resources", componentCtx.ValidateUpdateDeploymentsResources)
 	t.Run("Validate component disabled", componentCtx.ValidateComponentDisabled)
-	// t.Run("Validate component releases", componentCtx.ValidateComponentReleases)
+	t.Run("Validate component releases", componentCtx.ValidateComponentReleases)
```

Comment:
can this check be moved before  `Validate component disabled` ? so the component is not left running once the test ends

Reply:
Good catch thanks, fixed in latest commit